addons/hr_payroll/wizard/hr_payslip_employees.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo.tools.translate import _
+from odoo.exceptions import UserError
+
+
+class HrPayslipEmployees(models.TransientModel):
+
+    _name ='hr.payslip.employees'
+    _description = 'Generate payslips for all selected employees'
+
+    employee_ids = fields.Many2many('hr.employee', 'hr_employee_group_rel', 'payslip_id', 'employee_id', string='Employees')
+
+    @api.multi
+    def compute_sheet(self):
+        HrPayslip = self.env['hr.payslip']
+        HrPayslipRun = self.env['hr.payslip.run']
+        if self.env.context and self.env.context.get('active_id'):
```

Comment:
if self.env.context.get('active_id') is enought : not required to check...............